ctypesgen/__main__.py

```diff
@@ -38,18 +38,16 @@ def main(givenargs=None):
     )
```

Comment:
Ideally these changes would go into the other currently unmerged commits (that recently introduced these lines). Using git commit --amend and --rebase. That way the git blame/history points to the logical commit that introduced these changes.